### DISCUSSION TITLE
Email Options

### DISCUSSION THREAD

**Discussion by way0utwest** [dataplat/DataSaturdays#170] [Ideas]
What email options would you want? I'd think that we need these choices for sending an email:
- all attendees
- all attendees and speakers
- all speakers
- specific speaker
- all volunteers
- organizers

Anything else? I wouldn't think an individual needs to be contacted from the system, but perhaps.

**Comment by DrJekyll325** [comment]
- all sponsors
- specific sponsor

**Comment by tameraclark** [comment]
- All registrations (including waitlist, not going, attending)
- previous event registrations

I could see needing to email a single attendee about a prize, but I also feel that the emails shouldn't be hidden from the organizer. 

Scheduling of emails and using previous emails as a template is pretty important.

**Comment by Avahey** [comment]
All checked in attendees to either send a thank you post event to those who attended or email updates if anything happens midway through the event.
